.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - "[0-9].[0-9]+.x"
+  pull_request: {}
+jobs:
+  checks:
+    name: checks
+    runs-on: ubuntu-latest
+    steps:
+      # the skip-duplicate-actions step actually applies to the whole workflow
+      - uses: fkirc/skip-duplicate-actions@master
+        name: cancel previous runs
+        with:
+          github_token: ${{ github.token }}
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - uses: eskatos/gradle-command-action@v1
+        name: gradle
+        with:
+          arguments: check javadoc
```

Comment:
`javadoc` is probably not terribly interesting for the site and could be skipped